app/javascript/packs/fundraiser.js

```diff
@@ -75,6 +76,7 @@ window.mountFundraiser = function(root: string, data: MountFundraiserOptions) {
   dispatch({ type: 'set_recurring_defaults', payload: rDefault });
 
   dispatch({ type: 'set_direct_debit_only', payload: search.dd_only === '1' });
+  dispatch({ type: 'search_string_overrides', payload: search });
```

Comment:
@eyko Why do you use an action for this if it's only used when initializing the component?